dockerfiles/grist/ressources/custom.css

```diff
@@ -57,3 +58,35 @@
   background-color: #16b378;
   border-color: #16b378;
 }
+/* END Custom ACL colors */
+
+/* START Custom document card in home page colors */
+.test-dm-public {
+  --icon-color: #cecece !important;
+}
+
+.test-dm-pinned-initials {
+  border-color: #cecece !important;
+}
+/* END Custom document card in home page colors */
+
+/* START Custom tooltip colors */
+.test-tooltip {
+  background-color: rgba(255, 255, 255, 0.8) !important;
+  color: #666666 !important;
+}
+
+.test-tooltip-close [style*=CrossSmall]{
+  background-color: #929292;
+}
+/* END Custom tooltip colors */
+
+/* START Custom Fullframe Card View Colors */
+.test-close-button {
```

Comment:
This rule does not work on my local environment. STR:
1. Create a new doc;
2. add a dummy line;
3. Press space to open the card view → the button has still the original style;

Reply:
Also I still haven't the close button fixed in the Raw Data (*Données sources*) view:
![image](https://github.com/numerique-gouv/dockerfiles/assets/371705/dcf97d19-cee8-4ba1-8ce4-ecb08c10dae1)